src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransformer.cs

```diff
@@ -22,7 +22,7 @@
 
 [assembly: LoadableClass(typeof(void), typeof(FeatureContributionEntryPoint), null, typeof(SignatureEntryPointModule), FeatureContributionCalculatingTransformer.LoaderSignature)]
 
-namespace Microsoft.ML.Data
+namespace Microsoft.ML.Transforms
```

Comment:
Note, this is the only class that moved from ML.Data ->ML.Transforms. 

Everythign else was a sub-namespace of ML.Transforms or ML.Trainers

Reply:
Oh geez. That's good that you caught it!